webapp/src/components/layout/Layout.tsx

```diff
@@ -18,7 +18,7 @@ export default function Layout({ children }: Props) {
     else setScrollOk(true);
   }
   return <GlobalErrorBoundary>
-    <Header logo={logo} list={['Home', 'Serie', 'Movie']} scrollOk={scrollOk} />
+    <Header logo={logo} list={['Home', 'Serie', 'Movie', 'Genre']} scrollOk={scrollOk} />
```

Comment:
Mettre la liste en constante avec un joli nom

```suggestion
    <Header logo={logo} list={navItems} scrollOk={scrollOk} />
```